toolbox/certbot/cron_entry

```diff
@@ -1 +1 @@
-0 */12 * * * root /usr/airflow/atd-airflow/toolbox/certbot/renew.sh >> /var/log/airflow_cert_renewal.log
+0 */12 * * * root /usr/airflow/atd-airflow/toolbox/certbot/renew.sh >> /var/log/airflow_cert_renewal.log 2>&1
```

Comment:
I'm adding this to combine both `stderr` and `stdout` into one data stream to be logged.